src/python/pants/auth/cookies.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+
+from six.moves.http_cookiejar import LWPCookieJar
+
+from pants.subsystem.subsystem import Subsystem
+from pants.util.dirutil import safe_mkdir_for
+
+
+class BasicAuthException(Exception):
```

Comment:
Kill - this looks like a dup to `pants.auth.basic_auth.BasicAuthException` and is unused.

Reply:
Done.